client/state/selected-editor/actions.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import { EDITOR_TYPE_SET } from 'state/action-types';
+import 'state/data-layer/wpcom/sites/gutenberg';
```

Comment:
@dmsnell This doesn't work for some reason I can't find; the API call will not fire when dispatching the action (from the console), as if this import isn't here. Keeping it (I also noticed that a new build isn't triggered when I add/remove this line.) I'm sure I'm missing something obvious-ish here.